Octokit.Tests/Clients/UserEmailsClientTests.cs

```diff
@@ -1,14 +1,14 @@
-﻿using NSubstitute;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using NSubstitute;
 using Xunit;
 
 namespace Octokit.Tests.Clients
 {
     public class UserEmailsClientTests
     {
-        public class TheGetAllMethod
+        public class TheGetAllMethods
```

Comment:
Yeah, I changed it :+1: 
Find one more TheGetAllMethods in [GistsClientTests.cs](https://github.com/octokit/octokit.net/blob/615b6900b9cbcdb6fe6e647c245b77c3ef7f8b38/Octokit.Tests/Clients/GistsClientTests.cs#L27)